__tests__/e2e_tests.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { test, expect } from '@playwright/test'
```

Comment:
A playwright tesztekek lehetne egy `e2e` folderbe tenni, es a `jest.config.ts`-ben beallitani, hogy az `e2e` folder ignoralva legyen (https://jestjs.io/docs/configuration#testpathignorepatterns-arraystring) mert jelenleg a Jest le akarja futtatni mint unit tesztet.

Reply:
elvileg ezt is javítottam